src/utils/converters.ts

```diff
@@ -19,6 +19,8 @@ interface PrepareRowsData {
   action: (id: string) => void;
 }
 
+const CDN_URL = 'https://assets.cdn.io.italia.it/logos/organizations/';
```

Comment:
It's better to define this const in a more wide context. Define it in the `.env` file please using a more verbose name like `ENTITIES_LOGO_CND`